wcs-functions.php

```diff
@@ -49,20 +49,23 @@ function wcs_is_subscription( $subscription ) {
 }
 
 /**
- * A very simple check. Basically if we have ANY subscriptions in the database, then the user has probably set at
- * least one up, so we can give them the standard message. Otherwise
+ * Return true if there are any subscriptions in the database (active or inactive).
```

Comment:
Updated this comment to clarify purpose. Happy to reinstate the other details if it's useful. Reviewers please feedback to make this clear!